Make is broken in jsonnet v0.15.0

As discussed on Slack, using later versions of `jsonnet` breaks `make` in newer versions.

```
$ jsonnet --version
Jsonnet commandline interpreter v0.15.0
```

```
$ make
find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
	xargs -n 1 -- jsonnet fmt -n 2 --max-blank-lines 2 --string-style s --comment-style s -i
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
ERROR: unrecognized argument: -n
make: *** [Makefile:6: fmt] Error 123
```

The reason for this seems to be https://github.com/google/jsonnet/pull/656 - which introduced `jsonnetfmt` as a separate binary.

For Fedora users: don't use the `golang-github-google-jsonnet` package as it does not contain `jsonnetfmt`

The last broken step is "dashboards_out"  

```
RUNTIME ERROR: couldn't open import "grafonnet/grafana.libsonnet": no match locally or in the Jsonnet library paths.
	lib/../dashboards/elasticsearch.libsonnet:2:17-53	thunk <grafana>
	lib/../dashboards/elasticsearch.libsonnet:3:19-26	thunk <dashboard>
	lib/../dashboards/elasticsearch.libsonnet:643:7-16	object <dashboards>
	lib/dashboards.jsonnet:4:11-27	object <anonymous>
	During manifestation	
make: *** [Makefile:17: dashboards_out] Error 1
```
which I'm figuring out as we speak.

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-JSONNET_FMT := jsonnet fmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 
 all: fmt prometheus_alerts.yaml prometheus_rules.yaml dashboards_out lint
 

--- a/README.md
+++ b/README.md
@@ -9,9 +9,14 @@ A set of Grafana dashboard and Prometheus alerts for Elasticsearch deployed as p
 This repository has been heavily inspired by <https://github.com/kubernetes-monitoring/kubernetes-mixin>.
 Follow that repository for further information about deployment.
 
+### Prerequisites
+
+Make sure you have [jb](https://github.com/jsonnet-bundler/jsonnet-bundler/releases), [promtool](https://github.com/prometheus/prometheus/releases), [jsonnet](https://github.com/google/jsonnet/releases) as well as [jsonnetfmt](https://github.com/google/jsonnet/releases) installed.
+
 ### Quick and dirty hint
 
 ```
+$ jb install
 $ make clean && make
 
 # Prometheus rules and alerts are generated here:

